main.js

```diff
@@ -0,0 +1,236 @@
+const CurrentWeather = function () {
+
+  // Property name to be saved in local storage when a user picks a default location
+  const STORAGE_ID = 'Weather';
+
+  // Save the weather object, which contains both current and forecast data, in local storage
+  let saveToLocalStorage = function () {
+    localStorage.setItem(STORAGE_ID, JSON.stringify(weather));
+  }
+
+  // Get the weather object, which contains both current and forecast data, from local storage
+  let getFromLocalStorage = function () {
+    return JSON.parse(localStorage.getItem(STORAGE_ID) || '{}');
+  }
+
+
+  /*
+    Weather object will hold current weather data based on the user's query.
+    Object includes the current weather and a separate array to contain an object for
+    each of the five days in the forecast. Data is pulled from local storage, if a default
+    location has been set by the user.
+  */
+  let weather = getFromLocalStorage();
```

Comment:
Do you really want to store the forecast in local storage too?  The value you want to save is the city name for searching (and again, you should be storing the state as well so that when it refreshes you get the right place (see eval document).  Imagine the scenario of saving durham as your default place, then 2 days later you open the page and your forecast is 2 days old (since you pull it from the old value).  Don't cache things that need to be live.